src/services/ai_service.py

```diff
@@ -1,9 +1,11 @@
 from typing import List, Dict
 from unidiff import Hunk, PatchedFile
+
 from ..core.config import Config
 from ..core.models import PRDetails
```

Comment:
Cảnh báo:  Việc sử dụng `hasattr(Config, 'ANTHROPIC_API_KEY') and Config.ANTHROPIC_API_KEY` có thể dẫn đến lỗi `AttributeError` nếu `Config` chưa được khởi tạo hoặc không có thuộc tính `ANTHROPIC_API_KEY`.  Nên thêm kiểm tra xem `Config` đã được khởi tạo hay chưa trước khi truy cập thuộc tính của nó.  Thêm kiểm tra rủi ro này có thể ngăn ngừa lỗi không mong muốn trong quá trình chạy.